lib/actions/wallet.js

```diff
@@ -151,16 +154,29 @@ function selectWallet(walletId, type) {
       info = await multisig.getInfo(walletId);
       // multisig wallets only have default account
       accounts = ['default'];
+
+      if (options.proposals) {
+        dispatch(getMultisigProposals(walletId, { mtx: true }));
+      }
     } else {
       info = await wallet.getInfo(walletId);
       accounts = await wallet.getAccounts(walletId);
     }
 
+    // reset tabs in send/receive view
+    // so that different instances of the
+    // same view start at the first tab
+    if (options.resetSendTab) {
+      dispatch(selectTab('sendReceive', 0));
+    }
+
     dispatch(setAccounts(walletId, accounts, type));
     dispatch(setWalletInfo(walletId, info, type));
 
     // get info for each account
     // TODO: test with many accounts
+    // TODO: does this take into account
+    // multisig vs standard?
```

Comment:
you should be able to get this information about an account if there is an `m` and `n` not 1 of 1